lib/core/text/abstract_text.nit

```diff
@@ -305,20 +305,27 @@ abstract class Text
 		end
 	end
 
-	# Returns `true` if the string contains only Numeric values (and one "," or one "." character)
+	# Returns `true` if the string contains only numeric values (and one "," or one "." character)
 	#
 	#     assert "123".is_numeric  == true
 	#     assert "1.2".is_numeric  == true
+	#     assert "-1.2".is_numeric == true
 	#     assert "1,2".is_numeric  == true
 	#     assert "1..2".is_numeric == false
+	#     assert "1.23e-02".is_numeric == true
+	#     assert "".is_numeric == true
 	fun is_numeric: Bool
 	do
 		var has_point_or_comma = false
+		var e_index = -1
 		for i in [0..length[ do
 			var c = chars[i]
 			if not c.is_numeric then
 				if (c == '.' or c == ',') and not has_point_or_comma then
```

Comment:
I think this should be modified too, commas should not be supported (yeah, blame me for this, I was young and foolish when I implemented this)